compose/material/material/src/desktopMain/kotlin/androidx/compose/material/DesktopMenu.desktop.kt

```diff
@@ -115,12 +123,31 @@ fun DropdownMenu(
                 }
             },
         ) {
+            val focusManager = LocalFocusManager.current
+
             DropdownMenuContent(
                 expandedStates = expandedStates,
                 transformOriginState = transformOriginState,
-                modifier = modifier,
+                modifier = modifier.onKeyEvent {
```

Comment:
It would be great if we do the same for Context menu.

Reply:
Just to be sure, do you mean `ContextMenuArea` ?

Reply:
Yes